### PR TITLE
translate deleted.html, fix url for canada.css

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

**Comment by fanjinfei** [comment]
ready for review

**Comment by fanjinfei** [comment]
@wardi Travis doenload gcj failed. How to fix that?

**Comment by wardi** [comment]
I just restarted the build. If you don't see that button on builds try logging in with your github account. I may need to do something to add you to the travis team, I'm not sure

File: ckanext/canada/templates/public/base.html [hunk 1]
```diff
@@ -47,5 +47,5 @@
 
 {%- block custom_styles %}
   {{ super() }}
-  <link rel="stylesheet" href="/static/canada.css" type="text/css" />
+  <link rel="stylesheet" href="{{ h.url_for_static('/static/canada.css') }}" type="text/css" />
```
Comment (LaurentGoderre) [inline ckanext/canada/templates/public/base.html]: :+1: I did the same on my environment

**Pull request merged**